articles/150_ros_command_line_arguments.md

```diff
@@ -201,6 +202,25 @@ ros2 run some_package some_ros_executable --ros-args --disable-rosout-logs --dis
 
 Logging is fully enabled by default, thus `--enable-*` options are usually redundant unless a `--disable-*` option found earlier in the command line is being overridden.
 
+#### Enclave assignments
+
+Enclave assignment may be achieved using the `--enclave`/`-e` option.
+This option takes a single string `value` assignment statement, where `value` is where value is fully qualified enclave path used to locate the respective security artifacts within the configured keystore.
+
+As an example, to assign a enclave path `/foo/bar` one may execute:
```

Comment:
a enclave -> an enclave

(same multiple times throughout the PR)

Reply:
The repetition throughout the doc made me doubt :D

Reply:
Done: 134f3e1